src/Bucket/MetricOperation.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace HelloFresh\Stats\Bucket;
+
+
+use HelloFresh\Stats\Bucket;
+
+class MetricOperation extends \SplFixedArray
+{
+    const LENGTH = 3;
+
+    /**
+     * MetricOperation constructor.
+     *
+     * @param array $operations
+     */
+    public function __construct(array $operations = [])
+    {
+        parent::__construct(static::LENGTH);
+
+        // ensure that operations is not bigger than allowed
+        array_splice($operations, static::LENGTH);
+
+        foreach ($operations as $key => $value) {
+            $this->offsetSet($key, $value);
```

Comment:
Since array in PHP can be a hash this may fail in a nice way. Maybe add a unit test with `['a'=> 'a', 'b' => 'b', 'c' => 'c']` as the $operations?